libsql-server/src/namespace/meta_store.rs

```diff
@@ -343,6 +417,46 @@ impl MetaStore {
         self.inner.lock().configs.contains_key(namespace)
     }
 
+    pub async fn update_namespace_schema_and_config(
```

Comment:
I don't think this is necessary, the schema is part of the config, update the config should be sufficient, if updating the link is part of updating the config itself